src/marshaller/HipieDDL.js

```diff
@@ -766,6 +766,7 @@
         this.URL = dataSource.URL;
         this.databomb = dataSource.databomb;
         this.request = {};
+        this.isLoaded = false;
```

Comment:
I think this should mirror the _renderCount that widgets have - IOW rename to _loadedCount and increment it each time its loaded.